yabee_libs/egg_writer.py

```diff
@@ -764,15 +804,16 @@ def collect_poly_tref(self, face, attributes):
 
                         if tex_name not in textures:
                             textures.append(tex_name)
+                            
 
             else:
                 # The object has no material, that means it will get no textures
                 print("WARNING: Object", self.obj_ref.name, "has no material assigned!")
 
             # Store all textures
             for tex_name in textures:
-                if tex_name in USED_TEXTURES: # Make sure that  we'll have this texture in header
-                    attributes.append('<TRef> { %s }' % eggSafeName(tex_name))
+                #if tex_name in USED_TEXTURES: # Make sure that  we'll have this texture in header #todo:add this back once empties are added for PBR nodes
+                attributes.append('<TRef> { %s }' % eggSafeName(tex_name))
```

Comment:
Why was this check removed?  It causes YABEE to output `<TRef>`s in some cases for textures that weren't exported.  Especially when you have a texture set to None.

I'm going to add this check back in for now because it's causing issues for some people, but it would be good to have a fix that works for PBR nodes too.